composable/chain.json

```diff
@@ -118,19 +119,19 @@
       },
       {
         "name": "v5",
-        "tag": "v5.0.1",
-        "recommended_version": "v5.1.0",
+        "tag": "v5.2.0",
+        "recommended_version": "v5.2.0",
         "compatible_versions": [
-          "v5.1.0"
+          "v5.1.0",
+          "v5.2.0"
         ],
         "cosmos_sdk_version": "v0.47.3",
         "ibc_go_version": "v7.0.1",
         "consensus": {
           "type": "cometbft",
           "version": "0.37.1"
         },
-        "height": 1515288,
-        "proposal": 10,
+        "height": 1769900,
```

Comment:
The height property should reflect when v5 started, and not when v5.2.0 started.

Reply:
Closing as this has been resolved with https://github.com/cosmos/chain-registry/pull/2919